src/Stoa.ts

```diff
@@ -1,10 +1,15 @@
 import express from "express";
+import bodyParser from "body-parser";
 
 class Stoa {
     public stoa: express.Application;
 
     constructor () {
         this.stoa = express();
+        // parse application/x-www-form-urlencoded
+        this.stoa.use(bodyParser.urlencoded({ extended: false }))
```

Comment:
That's for the `height` parameter right ?

Reply:
I don't think so. 
That is Returns middleware that only parses  and Content-Type header matches and accepts only UTF-8 encoding